### ISSUE TITLE
Are these XSL sheets compatible with specification 2.3?

### ISSUE THREAD

**Issue by darcome** [kibook/S1000D-XSL-Stylesheets#3]
Hi everyone,

just a quick question: are these XSL sheets compatible with specification 2.3?

Thanks in advance!

**Comment by kibook** [comment]
I don't believe so, the [original project](https://github.com/smartavionics/S1000D-XSL-Stylesheets) seems to have been written with 4.x in mind. Support for the 2.3 element names probably wouldn't be too difficult to add to the stylesheets, but it would take some time to adapt all the templates for them.

Alternatively, you could create a stylesheet that transforms 2.3 elements to 4.x ones, e.g. by converting `<para0>`, `<subpara1>`, etc., to `<levelledPara>`, `<step1>`, `<step2>`, etc., to `<proceduralStep>`, and so on. It means having to get familiar with the newer issues of the specification, but it's probably how I would approach this problem, at least initially, until the stylesheets can be adapted to support older issues on their own.
